### PR TITLE
add topic line

### PR DESCRIPTION
**Pull request by serignebidev**
we found that we should add topic line 3 times

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # git-practicing
 readme file
+topic topic topic
```
Comment (serignebidev1) [inline README.md:3]: i believe it would be better with topic 2 times

**Comment by serignebidev1** [comment]
if you make that changes. i'll be happy to merge this.

**Comment by serignebidev** [comment]
i changed it to 2 times.

**Comment by serignebidev1** [comment]
merged

**Pull request merged**